app/models/manageiq/providers/openshift/container_manager/orchestration_stack.rb

```diff
@@ -0,0 +1,56 @@
+class ManageIQ::Providers::Openshift::ContainerManager::OrchestrationStack < ManageIQ::Providers::ContainerManager::OrchestrationStack
+  require_nested :Status
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ContainerManager"
```

Comment:
@moolitayer @simon3z Any suggestion regarding the file structures? Should the kube/openshift specific code be moved to its own provider directory?